package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "root",
+  "private": true,
+  "devDependencies": {
+    "lerna": "^3.16.4"
+  },
+  "scripts": {
+    "install": "npm run bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
+    "bump-version": "lerna version --tag-version-prefix=common-",
```

Comment:
I'm not entirely clear on how lerna versioning is working here -- and how it fits into our existing CI ecosystem...

Reply:
See my comment here: https://github.com/keymanapp/keyman/pull/2091#issuecomment-533351805